sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -48,6 +48,10 @@ public void setFingerprint(List<String> fingerprint) {
     this.fingerprint = fingerprint;
   }
 
+  public void addFingerprint(String fingerprint) {
```

Comment:
the API is `setFingerprint` and it takes the whole `List<String>`. AFAIK we're not offering an API to only append to it. When one sets the fingerprint, it;s the whole list.